### PR TITLE
Offline Mode: Remove Deprecated Code – P13

### PR DESCRIPTION
**Pull request by kean**
I used Periphery to identify the remaining unused bits – stuff that wasn't explicitly annotated but is unused now.

## To test:

- Smoke test the editor
- Open "Post List" / "Search" and ✅ verify that you can still search by tags (just start typing)

## Regression Notes
1. Potential unintended areas of impact: Post 
2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift [hunk 1]
```diff
@@ -1211,24 +1202,6 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
...
-        }
-
-        switch keyPath {
-        case BasePost.statusKeyPath:
-            if let status = post.status {
-                postEditorStateContext.updated(postStatus: status)
-            }
-        case #keyPath(AbstractPost.date_created_gmt):
-            let dateCreated = post.dateCreated ?? Date()
-            postEditorStateContext.updated(publishDate: dateCreated)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift:1243]: This was one more advantage of removing separate "Schedule" – no KVO.

**Review by staskus — APPROVED** [review]
Looks good. Only needs some fixes in `PublishSettingsViewControllerTests`.

I also never used [Periphery](https://github.com/peripheryapp/periphery). Useful 👍

**Comment by kean** [comment]
> Looks good. Only needs some fixes in PublishSettingsViewControllerTests.

Fixed.

> I also never used [Periphery](https://github.com/peripheryapp/periphery). Useful 👍

It's not bad. I've used it before in this project to remove a large amount of code. There is still a lot more to go. I'll do one more sweep later today.

**Pull request merged**